src/subclass/mod.rs

```diff
@@ -14,5 +14,4 @@ pub mod prelude {
     pub use super::io_stream::IOStreamImpl;
     pub use super::output_stream::OutputStreamImpl;
     pub use super::seekable::SeekableImpl;
-    pub use glib::subclass::prelude::*;
```

Comment:
Now this one here needs some discussions. Should we always re-export preludes of dependencies from our own preludes, or not?

It seems useful to do but then we should also do that for `gio::prelude`. Opinions?

Reply:
I always wondered about this... Is it better to reexport dependencies public interfaces allowing users to not have to add dependencies themselves or should we keep things as they are? I really don't know...

Reply:
Personally I think it's nicer to re-export public dependencies from the preludes. That way you can at least use all the base class' APIs directly. It makes peoples life easier and less unexpected: I'd expect to be able to use GObject API directly on a `gtk::Widget` at least.

Reply:
Well then, since I'm neutral, it sounds like we're more interested into reexporting. What do you think @EPashkin ?

Reply:
As we begin to remove reexport glib Error, Inhibit etc. IMHO better continue this way.

While don't add explicit dependency is simple, it can cause misunderstanding (these type is same?) etc.
So I prefer don't reexport types if possible.

Reply:
Do you have an example how it can cause misunderstanding in the case of the preludes where you never use the types directly, but only via traits?

My main argument here why this is different is that you never use the types directly, and from other languages you'd expect to be able to use the base class API of every object you have without having to import anything new and additional.

Reply:
Example of misunderstanding with types: https://github.com/gtk-rs/gtk/issues/897
I seems was too focused to this case and missed main point: it not types but traits.
I think that reexporting traits is good idea.

Reply:
Ok, then I'll do that consistently and update all the crates one after another in the next days :) Thanks!